lucille-examples/lucille-opensearch-ingest-example/conf/tika-config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<properties>
+    <parsers>
+      <parser class="org.apache.tika.parser.DefaultParser"/>
```

Comment:
Would we get any benefit from not including DefaultParser here? Or from including it but putting the RFC parser first?

Reply:
DefaultParser is needed for parsing the body, RFC parser is need for the rest of the metadata